hypertrace-trace-enricher/trace-reader/src/main/java/org/hypertrace/trace/reader/attributes/TraceAttributeReaderFactory.java

```diff
@@ -0,0 +1,12 @@
+package org.hypertrace.trace.reader.attributes;
+
+import org.hypertrace.core.attribute.service.cachingclient.CachingAttributeClient;
+import org.hypertrace.core.datamodel.Event;
+import org.hypertrace.core.datamodel.StructuredTrace;
+
+public class TraceAttributeReaderFactory {
```

Comment:
Does the factory need to be a class? It's not constructable on its own, so seems like an interface would do